src/coreComponents/finiteElement/elementFormulations/Qk_Hexahedron_Lagrange_GaussLobatto.hpp

```diff
@@ -162,107 +277,127 @@ class Qk_Hexahedron_Lagrange_GaussLobatto final : public FiniteElementBase
-   * @return The determinant of the parent/physical transformation matrix.
-   */
+
+/**
+ * @brief Calculate the shape functions derivatives wrt the physical
+ *   coordinates.
+ * @param q Index of the quadrature point.
+ * @param X Array containing the coordinates of the mesh support points.
+ * @param gradN Array to contain the shape function derivatives for all
+ *   support points at the coordinates of the quadrature point @p q.
+ * @return The determinant of the parent/physical transformation matrix.
+ */
   GEOS_HOST_DEVICE
   static real64 calcGradN( localIndex const q,
                            real64 const (&X)[numNodes][3],
                            real64 ( &gradN )[numNodes][3] );
-
-  /**
-   * @brief Calculate the shape functions derivatives wrt the physical
-   *   coordinates at a single point.
-   * @param[in] coords The parent coordinates at which to evaluate the shape function value
-   * @param[in] X Array containing the coordinates of the support points.
-   * @param[out] gradN Array to contain the shape function derivatives for all
-   *   support points at the coordinates of the quadrature point @p q.
-   * @return The determinant of the parent/physical transformation matrix.
-   */
+/**
+ * @brief Calculate the shape functions derivatives wrt the physical
+ *   coordinates at a single point.
+ * @param[in] coords The parent coordinates at which to evaluate the shape function value
+ * @param[in] X Array containing the coordinates of the support points.
+ * @param[out] gradN Array to contain the shape function derivatives for all
+ *   support points at the coordinates of the quadrature point @p q.
+ * @return The determinant of the parent/physical transformation matrix.
+ */
   GEOS_HOST_DEVICE
   static real64 calcGradN( real64 const (&coords)[3],
                            real64 const (&X)[numNodes][3],
                            real64 ( &gradN )[numNodes][3] );
 
-  /**
-   * @brief Calculate the shape functions derivatives wrt the physical
-   *   coordinates.
-   * @param q Index of the quadrature point.
-   * @param X Array containing the coordinates of the support points.
-   * @param stack Variables allocated on the stack as filled by @ref setupStack.
-   * @param gradN Array to contain the shape function derivatives for all
-   *   support points at the coordinates of the quadrature point @p q.
-   * @return The determinant of the parent/physical transformation matrix.
-   */
+/**
+ * @brief Calculate the shape functions derivatives wrt the physical
+ *   coordinates.
+ * @param q Index of the quadrature point.
+ * @param X Array containing the coordinates of the mesh support points.
+ * @param stack Variables allocated on the stack as filled by @ref setupStack.
+ * @param gradN Array to contain the shape function derivatives for all
+ *   support points at the coordinates of the quadrature point @p q.
+ * @return The determinant of the parent/physical transformation matrix.
+ */
```

Comment:
Be careful w.r.t. the indentation?

Reply:
done -- indentation is more consistent now